lib/virus_scanner.rb

```diff
@@ -9,7 +9,8 @@ class Error < StandardError; end
   class InfectedFile < StandardError; end
 
   def scan(file_path)
-    out_str, status = Open3.capture2e('govuk_clamscan', '--no-summary', file_path)
+    clamscan_path = AssetManager.govuk.clamscan_path
+    out_str, status = Open3.capture2e(clamscan_path, "--no-summary", file_path)
```

Comment:
Very minor: I think it would be better to keep the single quotes around `'--no-summary'`.